datamodel/src/main/java/com/hartwig/serve/datamodel/Country.java

```diff
@@ -0,0 +1,22 @@
+package com.hartwig.serve.datamodel;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Value.Immutable
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class })
+public abstract class Country {
+
+    @NotNull
+    public abstract String countryName();
+
+    @NotNull
+    public abstract Set<String> cities();
+
+    @Nullable
+    public abstract Map<String, Set<String>> hospitalsPerCity();
```

Comment:
Will every city always appear as a key in the "hospitals per city" map? 

If yes, you could remove the `cities` property since it is implied by the keyset of `hospitalsPerCity`

I assume no since `hospitalsPerCity` is nullable and maybe sometimes not filled in?

My feeling is that it would be cleaner to relate the 2 fields (every city will always be a key in the map, though the value could be empty (or even null))

Reply:
As discussed. Removed cities(), since every city always appears as a key in `hospitalsPerCity()`